src/bun.js/api/Timer.zig

```diff
@@ -130,7 +130,7 @@ pub const All = struct {
     pub fn onUVTimer(uv_timer_t: *uv.Timer) callconv(.C) void {
         const all: *All = @fieldParentPtr("uv_timer", uv_timer_t);
         const vm: *VirtualMachine = @alignCast(@fieldParentPtr("timer", all));
-        all.drainTimers(vm);
+        all.drainTimers(vm) catch return;
```

Comment:
What are we supposed to do when draining timers has a termination exception? At the very least, I think we need to uv_run_stop or there is some libuv function which says "please stop the event loop" and that needs to run here